src/server/security/index.md

```diff
@@ -0,0 +1,49 @@
+# Secure Mergin Maps installation
+
+::: warning
+This sections aims to provide some guidelines and a minimalistic example on how to secure a Mergin Maps deployment.
+
+Further security enhancements should be implemented by experts in accordance to cybersecurity policies in place.
+
+:::
+For security and privacy reasons Mergin Maps deployments should enable HTTPS secured connection via certificate file.
+
+We provide a template configuration file <GitHubRepo id="MerginMaps/server/blob/master/ssl-proxy.conf" desc="ssl-proxy.conf" />as base for your configuration.
+
+Let's have a quick look at the main sections:
+
+``` shell
+    server {
+        listen                  443 ssl;
+        server_name             merginmaps.company.com; # FIXME
+        client_max_body_size    4G;
+        ...
+```
+
+Here we enable SSL via the default `443` port and configure name-based HTTPS server via `server_name`. Here you should change this according to your target server name.
+
+We don't recommend setting a `client_max_body_size` higher than specified, because that might lead to timeouts while uploading your data to Mergin Maps.
+
+Next, you need to point your certificate files to NGINX configuration. This is done on the next lines on the secured configuration:
+
+``` shell
+    ...
+    ssl_certificate_key /etc/letsencrypt/live/merginmaps.company.com/privkey.pem; # FIXME
+    ssl_certificate     /etc/letsencrypt/live/merginmaps.company.com/fullchain.pem; # FIXME
+    ...
+```
+
+The above example uses automated keys generated by CertBot. For more information, visit [CertBot](https://certbot.eff.org/instructions) website and check how you can generate your own keys.
+
```

Comment:
maybe add some comment that several other security settings for headers should be reviewed

https://github.com/MerginMaps/server/blob/master/ssl-proxy.conf#L39 and below